src/web/source.rs

```diff
@@ -261,6 +261,7 @@ pub fn source_browser_handler(req: &mut Request) -> IronResult<Response> {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::test::*;
```

Comment:
Glob imports aren't the greatest, they obfuscate where things are coming from and make hidden dependencies on module layouts
```suggestion
    use crate::test::wrapper;
```

Reply:
Hmm, this has been the standard for all the tests (see e.g. https://github.com/rust-lang/docs.rs/blob/master/src/web/mod.rs#L580). The rationale is I want to make it as easy to add tests as possible without having to stop in the middle to add an import.

Reply:
Although I do see your point since this imported more than the wrapper:
```suggestion
    use crate::test::{wrapper, assert_success};
```

Reply:
Whoops, didn't see the `assert_success`. I just really don't like them, plus there's [others](https://github.com/rust-lang/docs.rs/blob/master/src/web/metrics.rs#L199) [that](https://github.com/rust-lang/docs.rs/blob/master/src/web/releases.rs#L684) [don't](https://github.com/rust-lang/docs.rs/blob/master/src/web/file.rs#L69)

Reply:
Ok, I removed the glob import.